tutorials/developer-tutorials/createNewIdentityAttributes.md

```diff
@@ -0,0 +1,72 @@
+## **Create New Identity With Attributes Using Local Fabric CA**
+`5-10 mins`
+
+> **Important:** You should have a **smart contract** instantiated on your Local Fabric to follow this tutorial. Follow part 1 of this series first for instructions.
+
+
+## Learning Objectives
+
+* Create a new identity (with attributes) using Local Fabric CA.
+* Using the newly created identity and implement attribute-based action control.
+
+---
+<details>
+<summary><b>1. Creating a new identity with attributes</b></summary>
+
+In the very first tutorial (Tutorial 1: Local smart contract development), we used a pre-set identity (`admin`) to connect to the Local Fabric gateway and submit/evaluate transactions. In this tutorial, we will learn how to create _our own_ identity with attributes.
+
+Having multiple identities enrolled with the Local Fabric CA (Certificate Authority) allows you to represent different participants in a blockchain network. For example two identities `Tesla` (the manufacturer) and `Jack` (the customer) may be permitted to submit different transactions in a smart contract.
+
+Created identities can be assigned attributes, which allows attribute-based access control. A transaction function could be written such that if the value of a given attribute on the identity being used matches a particular value, it will do one thing; if the value does not match, it could do something else, or throw an error. How to write such logic into smart contract transactions will be covered in a later tutorial.
+
+As an example, suppose we have a smart contract which allows participants to record the cars owned by a company on a blockchain. We may allow participants to add cars to the blockchain by submitting a `createCar()` transaction. Now, this would be fine for `Tesla` because it is a manufacturer, but since `Jack` is an individual, if he tries to submit a `createCar()` transaction, we should stop him! Here’s how you would register and enrol an identity for `Tesla` from the Local Fabric CA with suitable attributes…
+
+1. In the `FABRIC ENVIRONMENTS` panel, look for `ca.org1.example.com` (it's under Nodes), right click it and choose `Create Identity (register and enroll)`. 
```

Comment:
Again need to make sure they have the local fabric started and are connected

Reply:
Fixed.